markup.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Final Project: JiveKernel</title>
+    </head>
+    <body>
+        <img src='./JiveKernel-final-project.jpg' alt="wireframe" width="300">
+        <hr>
+        <h1>Final Project</h1>
+        <ul>
+            <li>Name: JiveKernel</li>
+            <li><p>Elevator pitch: A product for people who think LiveJournal is just too overburdened with features and users: A new journaling app for users that want a minimalistic feature set that no one much uses but which did help Fletcher figure out a bit of javascript programming.</p></li>
+            <li><p>Wireframe: (see picture above) Hitting the site without login state will present a login popup.  Once logged in, user will see a chronological list of entries of their own (and, if they have any, those of other users they've chose to friend).  Stretch goals for this project (as also pictured in the wireframe) will be the ability to search for entries and to comment on other users posts.</p></li>
+            <li>Dependencies:
+                <ul>
+                    <li>EC2 for hosting app</li>
+                    <li>Firebase</li>
+                </ul>
+            </li>
+            <li>Tasks: 
+                <ul>
+                    <li>Setup EC2/React</li>
+                    <li>Setup Firebase</li>
+                    <li>Create features:
+                        <ul>
+                            <li>Create Users</li>
+                            <li>Login/Logout</li>
+                            <li>Create Profile pages</li>
+                            <li>Add/Remove Friends</li>
+                            <li>Add/Edit/Remove Journal Entry</li>
+                            <li>Stretch goal: Comment on journal entries</li>
+                            <li>Stretch goal: Search for Journal entries</li>
```

Comment:
Sounds good 👍 Only recommendation from me would be to put some thought in how you organize the entries. You could organize entries under users, or have entries be its own top-level collection + filter by userId. Both are doable but I think the latter would be easier for displaying a collective list of entries.